runtime/reference/node.md

```diff
@@ -182,17 +162,18 @@ of moving components in your project. Deno also has a more secure runtime, with
 [runtime permissions](../basics/permissions.md) that allow you to control what
 your code can access.
 
-#### deno.json (optional)
+#### deno.json
```

Comment:
These were purposefully marked as optional to make it clear that they're not a required change to migrate from Node.

Reply:
It's strange because all of them have (optional) and they're already under a heading that days "optional". I can leave it but u might take another stab at restructuring it

Reply:
Here we should probably discuss package.json and deno.json - saying that dependencies can be defined in either deno.json's import map, or package.json dependencies, OR inline without either.